packages/design-system-scripts/gulp/docs/generatePages/generatePages.js

```diff
@@ -1,16 +1,33 @@
-const addReactDocs = require('./addReactDocs');
+// Run babel transforms on src files so we can run JSX scripts in Gulp tasks
+require('@babel/register')({
+  only: [/(@cmsgov\/design-system-docs|design-system\/packages\/([a-z-_]+)\/src|generateDocPage)/],
+  presets: [
```

Comment:
Presets and plugins are identical to the core CMSDS default babel.config.js, but we hardcode it so that customized babel configs in child design systems wont affect our doc site compiling